crates/router/src/services/api.rs

```diff
@@ -1865,9 +1869,9 @@ pub fn build_redirection_form(
     }
 }
 
-pub fn build_payment_link_html(
+pub fn build_payment_link_template(
```

Comment:
If this function returns the context (which I see as an intermediate result in the process of building the HTML), can we make this private?

Reply:
yes, we can make it private!